docs/tutorials/excel-tutorial-create-custom-functions.md

```diff
@@ -0,0 +1,203 @@
+* Office 2016 for Windows and [join the Office Insider program](https://products.office.com/office-insider). You must have Office build number 8711 or later.
+
+## Create your add-in project
+
+You’ll begin this tutorial by using the Yo Office Yeoman generator, which will automatically populate the files you need for your project.
+
+    1. In your command line interface, create a scaffold of your project (by default, this should be in your C:\Users\YourUserName folder).
+
+    ```bash
+    yo office
+    ```
+
+    ![Yo Office bash prompts for custom functions](../images/yo-office-excel-cfs-stock-ticker.png)
+    Answer the prompts as directed below:
+
+    * Choose a project type: `Excel Custom Functions Add-in project (September 2018 Preview Refresh: Requires the Insider channel for Excel)`
+    * What do you want to name your add-in? `stock-ticker`
+
+    For this tutorial, choose Javascript as the language you would like to use to build your add-in.
+
+    After you complete the wizard, the generator will create the project files and install supporting Node components.
+
+    2. Next, start a localhost instance by running one of the below commands in your command line interface.
+
+    If you are developing using the desktop version of Excel, use:
+
+    ```bash
+    npm start
+    ```
+
+    If you are using Excel Online, use:
+
+    ```bash
+    npm start-web
+    ```
+
+    3. You will also need to register your custom-functions add-in. In Excel, select **Insert > My Add-ins > Insert an Add-in**. This will bring up a list of available add-ins. Under "Developer Add-ins" you will see your add-in, under the name "Excel Custom Function". Select it to register it.
+
+    Select **Insert > Add-ins**. Choose **Manage My Add-ins** and select **Upload My Add-in**. Click "Browse..." for your manifest file (`C:\Users\LabUser\Stock Ticker\manifest.xml`), then click Open, select **Upload**. 
+
+    4. Finally, change the script tag to point to the right custom functions source. Open up your add-in project in your favorite code editor. In **index.html** in the root folder, delete and replace the script tag immediately following the <title> tags with the code below:
+
+    ```js
+    <script src="https://unpkg.com/@microsoft/office-js@1.1.9-adhoc.22/dist/custom-functions-runtime.js" type="text/javascript"></script>
+    ```
+
+## Try out a basic computational custom function
+
+Now the custom functions in your file will be loaded and ready to use. There are several pre-built functions for you in the Yo Office project. All are attached to a namespace called CONTOSO which is defined in the XML manifest file. Once you start typing =CONTOSO in a cell, the list of available functions will appear.
+
+Let's call =CONTOSO.ADD42(). This function adds 42 to any two numbers you provide as arguments. In any cell, type =CONTOSO.ADD42(1,2). It should deliver the answer 45.
```

Comment:
Please put the custom functions in back ticks like this: `=CONTOSO.ADD42()` throughout the article so they are formatted as code. Alternatively, format them in bold, since they are, in effect, part of the Excel UI.